_pages/en/open-source-code-schema.html

```diff
@@ -8,3 +8,300 @@
 <h1>Metadata Schema for Open Source Code</h1>
 <p>The metadata schema for the <a href="open-source-code.html">Open Source Code</a> section.</p>
 <p>Data files are hosted on <a href="https://github.com/canada-ca/ore-ero/tree/master/_data" target="_blank">GitHub</a>.
+<p>Mandatory fields are marked in dark blue.</p>
+
+
+<ul class="list-unstyled">
+    <li>
+    <details> 
+        <summary>Field Name</summary>
+        <p>Description</p> 
+    </details>
+    </li>  
+            
+    <li>
+    <details> 
+        <summary class="bg-primary">schemaVersion</summary>
```

Comment:
Putting the background on the `summary`, the color does not show when you click to expand the `details` and the focus is on it.

Maybe use a span, see example